azure/templates/run-tests.yml

```diff
@@ -96,6 +101,18 @@ steps:
           export INTEGRATION_API_KEY="$(INTEGRATION_API_KEY)"
           make install-python && ${{ parameters.test_command }}
         displayName: Run nightly test suite
+    - ${{ else }}:
+      - template: "azure/components/get-aws-secrets-and-ssm-params.yml@common"
+        parameters:
+          secret_file_ids:
+          - ptl/api-deployment/communications-manager/PRODUCTION_PRIVATE_KEY
```

Comment:
this needs to be `prod/api-deployment/communications-manager/PRODUCTION_PRIVATE_KEY` instead